python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/port-channel-interfaces.j2

```diff
@@ -649,6 +649,91 @@ interface {{ port_channel_interface.name }}
 {%     if port_channel_interface.isis_authentication_key is arista.avd.defined %}
```

Comment:
prioritise the new model, same as ethernet interfaces.

Reply:
Fixed. Thanks